Formula/d/dragonbox.rb

```diff
@@ -0,0 +1,46 @@
+class Dragonbox < Formula
+  desc "Reference implementation of Dragonbox in C++"
+  homepage "https://github.com/jk-jeon/dragonbox"
+  url "https://github.com/jk-jeon/dragonbox/archive/refs/tags/1.1.3.tar.gz"
+  sha256 "09d63b05e9c594ec423778ab59b7a5aa1d76fdd71d25c7048b0258c4ec9c3384"
+  license "Apache-2.0"
```

Comment:
```suggestion
  license any_of: ["Apache-2.0", "BSL-1.0"]
```

Reply:
make sense, updated to use `"Apache-2.0" => { with: "LLVM-exception" }` 👍

Reply:
Ah, nice catch, didn't notice that one!